src/logging.cpp

```diff
@@ -214,32 +229,31 @@ void BCLog::Logger::LogPrintStr(const std::string &str)
 
     m_started_new_line = !str.empty() && str[str.size()-1] == '\n';
```

Comment:
May I suggest to replace `str[str.size()-1]` with `str.back()`?

Reply:
Leaving it to minimize the diff